testsuite/features/secondary/proxy_retail_pxeboot_and_mass_import.feature

```diff
@@ -215,6 +215,19 @@ Feature: PXE boot a Retail terminal
     Then I should see a "1 package install has been scheduled" text
     When I wait until event "Package Install/Upgrade scheduled by admin" is completed
 
+  Scenario: Chect that the PXE boot minion stays configured after reboot
+    # salt key, channels and installed packages must survive reboot
+    When I reboot the PXE boot minion
+    And I wait for "60" seconds
```

Comment:
I don't like this, I would prefer to wait until some event happens, for instance waiting until the salt-minion service inside that VM is active, or something similar.

Example: 
`I wait until "salt-minion" service is active on "pxeboot_minion"`

Reply:
I guess you can refactor the step "completed during last minute", in order that in does not only check it's completed during last minute, but also checks it's the first row of the table. 🤔

Reply:
What about...
```
    And I wait until I see "Apply states [util.syncstates, saltboot]" text, refreshing the page
    And I follow first "Apply states [util.syncstates, saltboot]"
    And I wait at most 60 seconds until the event is completed, refreshing the page
```

Reply:
I am not a 100 % sure we are safe against race conditions.

For example, maybe the event is not in "Pending" yet, and you would go directly to "History". And then the first event is the old one.

Just saying. Maybe it's safe enough for our needs. We would need to try.

Reply:
If it fails, I come back to my first suggestion:
```
When I reboot the PXE boot minion
And I wait until "salt-minion" service is active on "pxeboot_minion"
Given I am on the Systems overview page of this "pxeboot_minion"
When I follow "Software" in the content area
And I follow "Software Channels" in the content area
Then radio button "Test-Channel-x86_64" is checked
```